Specs/Renderer/BufferSpec.js

```diff
@@ -11,22 +11,21 @@ defineSuite([
         createContext) {
     'use strict';
 
-    createBufferSpecs();
-
-    var c = createContext({ requestWebgl2 : true });
-    if (c.webgl2) {
-        createBufferSpecs({ requestWebgl2 : true });
-    }
-    c.destroyForSpecs();
+    createBufferSpecs({});
+    createBufferSpecs({
+        requestWebgl2 : true
+    });
 
     function createBufferSpecs(contextOptions) {
-        var context;
         var buffer;
         var buffer2;
+        var context = createContext(contextOptions);
```

Comment:
Creating the context here instead of `beforeAll` executes when the tests are created. Isn't it possible to lose the context before the tests run (because we create so many contexts)?